frontend/src/components/widgets/Slider/styled-components.ts

```diff
@@ -71,9 +71,12 @@ export const StyledTickBar = styled.div(({ theme }) => ({
   display: "flex",
 }))
 
-export const StyledTickBarItem = styled.div(({ theme }) => ({
-  lineHeight: theme.lineHeights.base,
-  fontWeight: "normal",
-  fontSize: theme.fontSizes.sm,
-  fontFamily: theme.fonts.monospace,
-}))
+export const StyledTickBarItem = styled.div<StyledSliderProps>(
+  ({ isDisabled, theme }) => ({
```

Comment:
nit: For consistency, it might be nicer to use `disabled` as prop name instead of `isDisabled`.